Objective-C/Tests/ReplicatorTest+Main.m

```diff
@@ -1824,6 +1826,28 @@ - (void) testReplicationConfigSetterMethods {
     AssertEqualObjects(headers, config.headers);
     AssertEqualObjects(docIds, config.documentIDs);
     AssertEqualObjects(channels, config.channels);
+    AssertEqual(config.heartbeat, 300);
+}
+
+- (void) testHeartbeatWithInvalidValue {
```

Comment:
This test also tests default heartbeat. Can we separate the tests or rename the test?

Reply:
Removed the default test.